### ISSUE TITLE
Security Checker not working via 1.2.x

### ISSUE THREAD

**Issue by stefandoorn** [Sylius/Sylius#10207]
**Sylius version affected**: 1.2.x

Sylius 1.2.x requires https://github.com/SyliusLabs/SensioDistributionBundle, which in turns requires the SensioLabs Security Checker in a 4.x version. This one stopped working due to change of hostnames. I would like to update Security Checker to 5.x, which doesn't work now:

```
composer require --dev sensiolabs/security-checker  --no-scripts --ignore-platform-reqs
Using version ^5.0 for sensiolabs/security-checker                                
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)                                     
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for sensiolabs/security-checker ^5.0 -> satisfiable by sensiolabs/security-checker[5.0.x-dev, v5.0.0, v5.0.1, v5.0.2, v5.0.3].
    - sylius-labs/sensio-distribution-bundle v6.0.1 requires sensiolabs/security-checker ~3.0|~4.0 -> satisfiable by sensiolabs/security-checker[4.1.x-dev].
    - sylius-labs/sensio-distribution-bundle v6.0.1 requires sensiolabs/security-checker ~3.0|~4.0 -> satisfiable by sensiolabs/security-checker[4.1.x-dev].
    - sylius-labs/sensio-distribution-bundle v6.0.1 requires sensiolabs/security-checker ~3.0|~4.0 -> satisfiable by sensiolabs/security-checker[4.1.x-dev].
    - Conclusion: don't install sensiolabs/security-checker 4.1.x-dev
    - Installation request for sylius-labs/sensio-distribution-bundle (locked at v6.0.1) -> satisfiable by sylius-labs/sensio-distribution-bundle[v6.0.1].


Installation failed, reverting ./composer.json to its original content.
```

But, I think it would be best if the DistributionBundle gets updated to support this out of the box in the next `composer update` any project user is doing. I can't file issues there, so I'm doing it here :-)

**Comment by stefandoorn** [comment]
Related change https://github.com/sensiolabs/security-checker/issues/149. I see the SensioDistributionBundle has a proper version in `composer.json`, but just no accompanying release.

**Comment by pamil** [comment]
Should be fixed by now, our fork of SensioDistributionBundle has been released and the changes the build fixed for Sylius 1.1+.

**Comment by stefandoorn** [comment]
👍 Can confirm it works.
